packages/manager-react-components/package.json

```diff
@@ -46,17 +44,19 @@
     "clsx": "^2.1.1",
     "lodash.isdate": "^4.0.1",
     "lodash.isequal": "^4.5.0",
-    "react": "^18.2.0",
     "react-use": "^17.5.0",
+    "react-i18next": "^14.0.5",
     "sass": "1.71.0",
     "tailwindcss": "^3.4.4",
     "uuid": "^9.0.1"
   },
   "devDependencies": {
     "@babel/core": "7.22.10",
     "@mdx-js/react": "^3.0.1",
+    "@ovh-ux/manager-core-api": "^0.9.0-alpha.0",
+    "@ovh-ux/manager-react-shell-client": "^0.8.0-alpha.0",
```

Comment:
```suggestion
```
it should be removed from devDependencies

Reply:
@frenautvh it is required for successful "tsc" https://github.com/ezolenko/rollup-plugin-typescript2/issues/277#issuecomment-1112504639

Reply:
whats can be the solutionto not duplicate theses dependencies as we already have "prepare": "tsc" command in "@ovh-ux/manager-core-api" and "@ovh-ux/manager-react-shell-client" ?

Reply:
or in pci common we dont have prepare comand